src/test/runner.ts

```diff
@@ -93,17 +100,25 @@ export class Runner {
       await this._flushOutput();
       return 'failed';
     }
-    if (result === 'forbid-only') {
+    if (result?.status === 'forbid-only') {
       console.error('=====================================');
       console.error(' --forbid-only found a focused test.');
+      for (const testName of result?.testNames)
+        console.error(` -- ${testName}`);
       console.error('=====================================');
-    } else if (result === 'no-tests') {
+    } else if (result!.status === 'no-tests') {
       console.error('=================');
       console.error(' no tests found.');
       console.error('=================');
+    } else if (result?.status === 'clashing-spec-titles') {
+      console.error('=================');
+      console.error(' tests with the same name per Suite are not allowed.');
```

Comment:
`per Suite` is not clear. Let's just say "duplicate test titles are not allowed"?